frame/clearing-house/.pre-commit-config.yaml

```diff
@@ -6,3 +6,9 @@ repos:
     entry: rustfmt --config skip_children=true --check
     language: system
     types: [rust]
+- repo: https://github.com/alessandrojcm/commitlint-pre-commit-hook
+  rev: v8.0.0
+  hooks:
+  - id: commitlint
+    stages: [commit-msg]
+    additional_dependencies: ['@commitlint/config-conventional']
```

Comment:
Is it required to have internet access every time one would commit?
(How does this remote repo reference works?)

Reply:
Neat. Thanks for the info.